paasta_tools/cli/cmds/spark_run.py

```diff
@@ -350,6 +350,13 @@ def add_subparser(subparsers):
         default=False,
     )
 
+    list_parser.add_argument(
+        "--uses-bulkdata",
+        help="Mount /nail/bulkdata in the container",
+        action="store_true",
+        default=False,
```

Comment:
Should we not set the default to true for now, then roll out my change to add the flag everywhere, and then set the default to false?

Reply:
hmm I checked using the following command and it seems `/nail/bulkdata` isn't mounted:
```bash
paasta spark-run --aws-profile=dev --cmd bash
```